calculate_documentation_coverage.py

```diff
@@ -44,7 +44,8 @@ def parse_files(directory):
     return public_methods, documented_methods
 
 # Specify the directory containing your Python files
-directory = "../src/qudi/"
+directory = os.path.join("src", "qudi")
```

Comment:
I don't think this is equivalent to what it was before since it's missing the `../`. Could you check this?

Reply:
Yes, I know. But it actually turned out to be that the one without the ../ is the correct one so that's why i made it like this.